### PR TITLE
feat: Implemented Relax Screen

### LINKED ISSUE #137: Implement Relax Screen with full Working
**Actual Behaviour**
Not Exist yet.

**Desktop Application Behaviour**
desktop-app-relax screen

.

**Expected behavior with Screenshot**

The parallax and moving effect of the space background and two _yantras_ images rotating and one is exactly top of other inside a full screen activity.
Relax Screen Github

**Would you like to work on the issue?**

Yes, I would love to work on this issue.

### PR DESCRIPTION
**Pull request by vishal-sehgal**
Fixes #137 

**Changes**
Made new screen and added 2 Yantras with rotating animation as per the desktop application.

**Screenshot/s for the changes**: 
relax-screen-neurolab

**Checklist**: 
- [x] I have used resources from `strings.xml`, `dimens.xml` and `colors.xml` without hard-coding them
- [x] No modifications done at the end of resource files `strings.xml`, `dimens.xml` or `colors.xml`
- [x] I have reformatted code in every file included in this PR [<kbd>CTRL</kbd>+<kbd>ALT</kbd>+<kbd>L</kbd>]
- [x] My code does not contain any extra lines or extra spaces
- [x] I have requested reviews from other members

**APK for testing**: 
[Neurolab-Android.zip](https://github.com/fossasia/neurolab-android/files/3045839/Neurolab-Android.zip)

### CODE REVIEW

**Comment by vishal-sehgal** [comment]
@iamareebjamal @jddeep Can you please Review it and If found any changes then please let me know.

**Comment by vishal-sehgal** [comment]
@jddeep Have you checked this buddy?

File: app/src/main/res/layout/activity_program_mode.xml [hunk 1]
```diff
@@ -1,32 +1,13 @@
...
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
     android:background="@drawable/hill_background"
     tools:context=".ProgramModeActivity">
 
-    <RelativeLayout
-        android:layout_width="match_parent"
-        android:layout_height="match_parent">
+    <include layout="@layout/layout_focus_screen" />
```
Comment (jddeep) [inline app/src/main/res/layout/activity_program_mode.xml]: @CoderVishalSehgal I think we should make different program mode activities rather than just toggling the visibilities of them (their layouts) in a single parent activity layout. @iamareebjamal What do you think?

**Comment by jddeep** [comment]
And @CoderVishalSehgal Good use of the`rotate` animation. Looks great! 👍

**Comment by vishal-sehgal** [comment]
@iamareebjamal Please merge this PR.

**Review by iamareebjamal — DISMISSED** [review]

**Comment by iamareebjamal** [comment]
Resolve Conflicts

**Comment by vishal-sehgal** [comment]
@iamareebjamal Resolved conflicts successfully.

**Comment by vishal-sehgal** [comment]
@iamareebjamal Please have a look.

File: app/src/main/res/layout/activity_program_mode.xml (hunk 1 above)
Comment (iamareebjamal) [reply app/src/main/res/layout/activity_program_mode.xml]: Use Fragment

File: app/src/main/java/io/neurolab/ProgramModeActivity.java [hunk 2]
```diff
@@ -65,6 +75,21 @@ protected void onCreate(Bundle savedInstanceState) {
                 setTitle(R.string.serial);
                 break;
         }
+    }
+
+    private void setupFocusScreen() {
```
Comment (iamareebjamal) [inline app/src/main/java/io/neurolab/ProgramModeActivity.java]: Use separate Fragments please

**Review by iamareebjamal — CHANGES_REQUESTED** [review]

File: app/src/main/java/io/neurolab/ProgramModeActivity.java (hunk 2 above)
Comment (vishal-sehgal) [reply app/src/main/java/io/neurolab/ProgramModeActivity.java]: Okay, then I will use fragments here.

**Comment by vishal-sehgal** [comment]
@iamareebjamal Implemented Fragments successfully for the _Relax_ and _Focus_ Visuals Screen. Please have a look.

File: app/src/main/java/io/neurolab/fragments/FocusVisualFragment.java [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+
+import io.neurolab.R;
+import io.neurolab.visuals.SpaceAnimationVisuals;
+
+/**
+ * A simple {@link Fragment} subclass.
+ */
```
Comment (iamareebjamal) [inline app/src/main/java/io/neurolab/fragments/FocusVisualFragment.java]: Useless comment

File: app/src/main/java/io/neurolab/fragments/FocusVisualFragment.java [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+    private View mView;
+
+
+    private ImageView rocketimage;
+    private int lastPos = 0;
+    private int newPos = -300;
+    private boolean moving = false;
+
+    public FocusVisualFragment() {
```
Comment (iamareebjamal) [inline app/src/main/java/io/neurolab/fragments/FocusVisualFragment.java]: Useless constructor

File: app/src/main/java/io/neurolab/fragments/FocusVisualFragment.java [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+
+import io.neurolab.R;
+import io.neurolab.visuals.SpaceAnimationVisuals;
+
+/**
+ * A simple {@link Fragment} subclass.
+ */
+public class FocusVisualFragment extends android.support.v4.app.Fragment {
+
+    private View mView;
```
Comment (iamareebjamal) [inline app/src/main/java/io/neurolab/fragments/FocusVisualFragment.java]: We don't use hungarian notation

File: app/src/main/java/io/neurolab/visuals/SpaceAnimationVisuals.java [hunk 6]
```diff
@@ -34,7 +34,7 @@ public void onAnimationEnd(Animation animation) {
 
                 @Override
                 public void onAnimationRepeat(Animation animation) {
-                   // required to repeat the Animation if user wants to see again
+                    // required to repeat the Animation if user wants to see again
```
Comment (iamareebjamal) [inline app/src/main/java/io/neurolab/visuals/SpaceAnimationVisuals.java]: Unnecessary change

File: app/src/main/java/io/neurolab/visuals/SpaceAnimationVisuals.java [hunk 7]
```diff
@@ -44,4 +44,4 @@ public void onAnimationRepeat(Animation animation) {
             view.startAnimation(launch);
         }
     }
-}
+}
```
Comment (iamareebjamal) [inline app/src/main/java/io/neurolab/visuals/SpaceAnimationVisuals.java]: Unnecessary change

File: app/src/main/res/values/strings.xml [hunk 8]
```diff
@@ -43,4 +43,4 @@ A difference to existing projects like OpenBCI is that it will not be necessary
     <string name="device_is_not_connected">Device is not connected</string>
     <string name="set_up">Set up</string>
 
-</resources>
+</resources>
```
Comment (iamareebjamal) [inline app/src/main/res/values/strings.xml]: Unnecessary change

File: app/src/main/java/io/neurolab/fragments/FocusVisualFragment.java (hunk 5 above)
Comment (vishal-sehgal) [reply app/src/main/java/io/neurolab/fragments/FocusVisualFragment.java]: Oops, I forget about it, will change it.

File: app/src/main/res/values/strings.xml (hunk 8 above)
Comment (vishal-sehgal) [reply app/src/main/res/values/strings.xml]: I didn't change a line there but don't know why it is showing this 😐

File: app/src/main/java/io/neurolab/ProgramModeActivity.java [hunk 9]
```diff
@@ -38,25 +33,34 @@
...
+        requestWindowFeature(Window.FEATURE_NO_TITLE);
+        getWindow().setFlags(WindowManager.LayoutParams.FLAG_FULLSCREEN,
+                WindowManager.LayoutParams.FLAG_FULLSCREEN);
         setContentView(R.layout.activity_program_mode);
 
-        rocketimage = findViewById(R.id.rocketimage);
 
         Intent intent = getIntent();
         Bundle bundle = intent.getExtras();
+        assert bundle != null;
```
Comment (iamareebjamal) [inline app/src/main/java/io/neurolab/ProgramModeActivity.java]: Remove

File: app/src/main/res/values/colors.xml [hunk 10]
```diff
@@ -9,4 +9,6 @@
     <color name="color_white_background">#fff</color>
     <color name="home_activity_background">#F2F2F2</color>
     <color name="seperator_color">#707070</color>
+    <color name="yantraOneColor">#FFF852</color>
```
Comment (iamareebjamal) [inline app/src/main/res/values/colors.xml]: Use underscore

**Comment by vishal-sehgal** [comment]
@iamareebjamal Please review and merge it.

**Review by iamareebjamal — APPROVED** [review]
